e2e/specs/add-note.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test } from '../core';
+import { expect } from '@playwright/test';
+import { NotesPage } from '../pages';
+import { deletePatient, generateRandomPatient, Patient } from '../commands';
+
+let patient: Patient;
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('should be able to add a note to a patient', async ({ page, api }) => {
+  const notesPage = new NotesPage(page);
+
+  await test.step('When I visit the notes page', async () => {
```

Comment:
@RandilaP, We might need to update the gherkin syntax keywords of theses steps. Please refer to [this tutorial](https://www.guru99.com/gherkin-test-cucumber.html#4). 

Here's a small overview of how to use the Gherkin keywords:
<img width="760" alt="image" src="https://github.com/openmrs/openmrs-esm-patient-chart/assets/27498587/d54ff7b6-9906-4ae5-91b6-65000f006a93">

Reply:
We use the "And" keyword when 2 or more consecutive steps with the same starting word come up.

Example:
```
When I input the email address
When I input the correct password
```

The above can be replaced with:
```
When I input the email address
And I input the correct password
```